### DISCUSSION TITLE
TypeError: vn[a[0]] is not a function

### DISCUSSION THREAD

**Discussion by chris2733** [avitorio/outstatic#92] [Q&A] [answered]
Having an issue getting setup with nextjs and typescript, going to /outstatic works just fine, but when I login I get this error popping up:

<img width="584" alt="image" src="https://user-images.githubusercontent.com/4907924/221363567-b0c3495f-bbb4-4b53-94b7-84c3ce958943.png">

As far as I can tell the env is correct and so is my auth, I'm not too sure where to go from here.

**Comment by avitorio** [comment, marked as answer]
Hey @chris2733,

Thanks for bringing this up. Would it be possible for you to clone your setup on a public repo so we can test it? Thanks.

**Reply by chris2733** [reply]
Thanks! I set it up here if this helps: https://github.com/chris2733/next-portfolio/tree/netlify-cms-hookup Let me know if you need anything else

**Reply by avitorio** [reply]
Hey,
Please, try upgrading Next.js to 12.2.5 or above.

**Reply by chris2733** [reply]
Hey, I'm using Next version 12.3.1?

**Reply by avitorio** [reply]
Sorry, my bad. Thought you were on 12.1.3.
Apparently you missconfigured your `/api` folder.
It should be `/api/outstatic/[[...ost]].tsx`

**Reply by avitorio** [reply]
Let me know if this works :)

**Reply by chris2733** [reply]
Sorry I was off for a few days - yes that works! My bad, thank you so much for your patience :)
